manager/inhibitor.go

```diff
@@ -24,19 +26,19 @@ type InhibitRules []*InhibitRule
 type InhibitRule struct {
 	SourceFilters Filters
 	TargetFilters Filters
-	MatchOn       []string
+	MatchOn       model.LabelNames
 }
 
-// Returns those target AlertLabelSets which are not inhibited by any of the
-// source AlertLabelSets.
-func (i *InhibitRule) Filter(s AlertLabelSets, t AlertLabelSets) AlertLabelSets {
+// Returns those target []model.LabelSet which are not inhibited by any of the
```

Comment:
This comment reads funny now with naive substitution. Maybe just "label sets" instead of the type name?